BOOKBBANG/BOOKBBANG/Views/Home/SortingBookPicker.swift

```diff
@@ -0,0 +1,40 @@
+//
+//  SortingBookPicker.swift
+//  BOOKBBANG
+//
+//  Created by Jerrie on 7/31/24.
+//
+
+import SwiftUI
+
+
+enum SortBookBy: String, CaseIterable {
+    case Added = "최근 등록 순"
+    case Read = "최근 읽은 순"
+    case AddedPhrase = "구절 등록 순"
```

Comment:
```suggestion
    case recentRegister = "최근 등록 순"
    case recentRead = "최근 읽은 순"
    case registerPhrase = "구절 등록 순"
```

더욱 직관적인 네이밍으로 제안드려봅니다.